Solution1/BusinessLayer/Extension/WeatherDtoExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BusinessLayer.DTOs;
+
+namespace BusinessLayer.Extension
+{
+    public static class WeatherDtoExtensions
+    {
+        public static string PrintToString (this WeatherDTO weatherDTO)
+        {
+            return $"In {weatherDTO.CityName} {weatherDTO.Temp} C. {weatherDTO.Comment}";
+        }
+
+        public static WeatherDTO GetCommentByTemp(this WeatherDTO weatherDTO)
```

Comment:
better to name FillComment, cause you not return Comment here

Reply:
+